app/controllers/reimbursements_controller.rb

```diff
@@ -38,6 +38,11 @@ def reimbursement_params
   end
 
   def fetch_reimbursements
-    policy_scope(CaseContact.joins(:casa_case), policy_scope_class: ReimbursementPolicy::Scope)
+    case_contacts = CaseContact.joins(:casa_case).includes(
+      :creator,
+      :case_contact_contact_type,
+      contact_types: [:contact_type_group]
+    ).preload(:casa_case)
```

Comment:
It seems really nice, I'm just wondering if fetching `:casa_case` with `includes` won't be better 🤔 . There is some special reason for using `joins + preload`

Reply:
The thing is `joins + includes` with `:casa:case` fires only one huge query to fetch all the associated data(1 inner join, 4 outer join), while `preload + joins` keeps separate queries to load each association. In terms of database performance I tend to prefer several small queries to a huge one.

Reply:
Nice, thanks 😄